src/webhook/mutator.go

```diff
@@ -20,13 +30,19 @@ type BaseRequest struct {
 	DynaKube dynatracev1beta1.DynaKube
 }
 
+// MutationRequest contains all the information needed to mutate a pod
+// It is meant to passed into each mutator, so that they can mutate the elements in the way they need to,
+// and after passing it in to all the mutator the request will have the final state which can be used to mutate the pod.
 type MutationRequest struct {
 	*BaseRequest
 	Context          context.Context
 	Namespace        corev1.Namespace
 	InstallContainer *corev1.Container
 }
 
+// ReinvocationRequest contains all the information needed to reinvoke a pod
+// It is meant to passed into each mutator, so that they can mutate the elements in the way they need to,
```

Comment:
Maybe a annoying nitpick but: did you mean to write "It is meant to be passed...."?